apollo-router/src/configuration/experimental.rs

```diff
@@ -0,0 +1,103 @@
+use std::collections::HashMap;
+
+use serde_json::Value;
+
+pub(crate) fn print_all_experimental_conf() {
+    let available_exp_confs = serde_json::from_str::<HashMap<String, String>>(include_str!(
+        "../../experimental_features.json"
+    ));
+    match available_exp_confs {
+        Ok(available_exp_confs) => {
+            let available_exp_confs_str: Vec<String> = available_exp_confs
+                .into_iter()
+                .map(|(used_exp_conf, discussion_link)| {
+                    format!("\t- {used_exp_conf}: {discussion_link}")
+                })
+                .collect();
+            println!(
+                "List of all experimental configurations with related GitHub discussions:\n\n{}",
+                available_exp_confs_str.join("\n")
+            );
+        }
+        Err(err) => {
+            eprintln!(
```

Comment:
I think this should panic rather than print a warning. We have a malformed json file.

Reply:
I just prefer put this kind of data in a specific file to not put some weird json directly in our source code. I also prefer write JSON in a proper .json file with syntax highlighting from my code editor, I feel better :p 
There is no other reason TBH